tests/Behat/features/preview-a-block.feature

```diff
@@ -0,0 +1,44 @@
+Feature: Preview a non-inline-editable block
+  As an author
+  I want to preview the parent page of the non-inline-editable block I'm editing in the CMS
+  So that I can see how it would look like to my visitors
+
+  Background:
+    Given I add an extension "DNADesign\Elemental\Extensions\ElementalPageExtension" to the "Page" class
+      And content blocks are not in-line editable
+      And the page template includes elemental blocks
+      And a "page" "Blocks Page" with a "Alice's Block" content element with "Alice's content" content
+
+    Given I am logged in with "ADMIN" permissions
+      And I go to "/admin/pages"
+      And I left click on "Blocks Page" in the tree
+      And I set the CMS mode to "Edit mode"
+    Then I should see an edit page form
+      And I should see a list of blocks
+      And I should see block 1
+
+    Given I click on block 1
+    Then I should see "Alice's Block"
+      And the "Content" field should contain "Alice's content"
+
+  @javascript
+  Scenario: I can show a preview of the current block's page from the pages section
+    When I set the CMS mode to "Preview mode"
+    Then I can see the preview panel
+      And the preview contains "Alice's content"
```

Comment:
Indentation is wonky

Reply:
Can you be more specific? Indentation in features are inconstent between modules and often between features in a given module, so it's not super clear what the indentation _should_ be - seems to be that "And" should be indented, as it's adding on top of a when/given/then step.

Reply:
Oh right, I never noticed how inconsistent it is

Probably doesn't matter then.  Personally I'd just left align everything, with a 2 space indent after each Background or Scenario, though I guess it doesn't really matter since there's no standard